### DISCUSSION TITLE
Lesson 7 : I did remapping for @chainlink but "File not found" yet

### DISCUSSION THREAD

**Discussion by ana-creator** [Cyfrin/foundry-full-course-cu#467] [Q&A] [answered]
Hello!
I did remapping for @chainlink but "File not found" yet. I can not find the mistake! File AggregatorV3Interface.sol exists in local lib/chainlink-brownie-contracts/contracts.
I will be really appreciative if you can advise me on something!
![TOML](https://github.com/Cyfrin/foundry-full-course-f23/assets/59834914/c3bab1be-eaaa-4aa8-815f-5c7606d18d81)

**Comment by pratokko** [comment, marked as answer]
Hello , you can remove the  / after contracts  use this instead 

```
remappings = ["@chainlink/contracts=lib/chainlink-brownie-contracts/contracts/"]
```

**Reply by ana-creator** [reply]
Thank you very much! Thanks to you, I noticed that instead of remappings I used remapping. Now everything works! Thanks again!

**Reply by LeeJianXing** [reply]
me too, i was also using 'remapping' instead of 'remappings'.I change it to remappings and it compile with no error ;D
